tests/tests_lvm_auto_size_cap.yml

```diff
@@ -0,0 +1,89 @@
+---
+- hosts: all
+  become: true
+
+  tasks:
+    - include_role:
+        name: linux-system-roles.storage
+
+    - include_tasks: get_unused_disk.yml
+      vars:
+        min_size: 10g
+        max_return: 1
+
+    - command: lsblk -b -l --noheadings -o NAME,SIZE
+      register: storage_test_lsblk
```

Comment:
Will adding `changed_when: false` resolve this?

Reply:
yes - please add `changed_when: false`